mvpoly/tests/monomials.rs

```diff
@@ -381,6 +381,36 @@ fn test_mvpoly_compute_cross_terms_degree_six() {
     assert_eq!(lhs, rhs);
 }
 
+// The cross-terms of a sum of polynomials is the sum of the cross-terms, per
+// power.
+#[test]
+fn test_mvpoly_pbt_cross_terms_addition() {
```

Comment:
I'd check p_1 + lambda p_2 to check the full linearity property

Reply:
Good idea!

Reply:
Well no. It isn't true.